source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,124 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v5.37. 
 
 Latest Mattermost Releases:
+- [Release v7.0 - Major Release](#release-v7-0-major-release)
 - [Release v6.7 - Feature Release](#release-v6-7-feature-release)
 - [Release v6.6 - Feature Release](#release-v6-6-feature-release)
 - [Release v6.5 - Feature Release](#release-v6-5-feature-release)
-- [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
 
+## Release v7.0 - [Major Release](https://docs.mattermost.com/upgrade/release-definitions.html#major-release)
+
+**Release Day: 2022-06-15**
+
+### Important Upgrade Notes
+ - MySQL self-hosted customers may notice the migration taking longer than usual when having a large number of rows in the ``FileInfo`` table. For MySQL, it takes around 19 seconds for a table of size 0.7M rows. The time required for PostgreSQL is negligible.
+ - When a new configuration setting ``EnableAppBar`` is enabled, all channel header icons registered by plugins will be moved to the new App Bar, even if they do not explicitly use the new registry function to render a component there. The setting ``EnableAppBar`` defaults to ``false`` for self-hosted deployments.
+ - The value of ``ServiceSettings.TrustedProxyIPHeader`` defaults to empty from now on. A previous bug prevented this from happening in certain conditions. Customers are requested to check for these values in their config and set them to nil if necessary.
+ - **IMPORTANT:** Session length configuration settings have changed from days to hours. If anything goes awry during a server upgrade, it could result in users not being able to log in. For customers who use environment variables for these config settings, the automatic migration won't work, and new configuration  environment variables must be added for: ``MM_SERVICESETTINGS_SESSIONLENGTHWEBINHOURS=``, ``MM_SERVICESETTINGS_SESSIONLENGTHMOBILEINHOURS=``, and ``MM_SERVICESETTINGS_SESSIONLENGTHSSOINHOURS=``. The values need to be 24x the existing values in days. For example, if ``MM_SERVICESETTINGS_SESSIONLENGTHWEBINDAYS=30``, they should set ``MM_SERVICESETTINGS_SESSIONLENGTHWEBINHOURS=720``.
```

Comment:
1 - If this is important, let's list it first
2 - What does the following sentence mean? If the upgrade process breaks? Or something else?

```
If anything goes awry during a server upgrade, it could result in users not being able to log in.
```

Reply:
1. Done
2. @mkraft to confirm

Reply:
@jasonblais Yes, if users can't login or are logged out too soon. That would be a more straightforward way to write it.

Reply:
cc @cwarnermm ^